pushy/src/main/java/com/eatthepath/pushy/apns/PushType.java

```diff
@@ -139,7 +139,22 @@ public enum PushType {
      * @see ApnsClientBuilder#setSigningKey(ApnsSigningKey)
      * @see DeliveryPriority
      */
-    LOCATION("location");
+    LOCATION("location"),
+
+    /**
+     * <p>Indicates that a push notification is intended to update a running Live Activity. According to Apple's
+     * documentation:</p>
+     *
+     * <blockquote><p>Set the apns-topic header field of the request you sent to APNs using the following format:
+     * {@code <your bundleID>.push-type.liveactivity.}</p>
+     *
+     * <p>To update a Live Activity, set the value for the payload’s event key to update. To end a Live Activity,
+     * set it to end. If you end a Live Activity, include the final content state to make sure the
+     * Live Activity displays the latest data after it ends.</p></blockquote>
+     *
+     * @since 0.15.2
```

Comment:
I respect that you're following the pattern we set in the Javadoc for other fields in this enumeration, but I don't think the pattern quite fits in this case. For everything else, we're able to quote directly from https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns to explain the _purpose_ of each push type. Frustratingly, Live Activity push notifications still aren't documented there, so we can't quote it directly. Beyond that, the quote here doesn't really explain what this push type is for. It _does_ explain other important details about topics ad payloads, but I don't think this is (structurally!) where I'd expect to find those details.

So, in short, I think there are a few issues here:

1. We can't directly quote a parallel explanation for what this `push-type` is for
2. We're actually citing a _different_ set of documentation
3. There are other details in here that don't fit the structural pattern established for other keys

So with all that in mind, I'd propose some rephrasing:

```suggestion
     * Indicates that a push notification is intended to update a running Live Activity. Note that Live Activity updates
     * must be sent to a specific topic ({@code [base bundle ID].push-type.liveactivity}) and include specific payload
     * keys (see [insert links here when we sort out what's happening with payload builders]).
     *
     * @since 0.15.2
     *
     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
     *     Updating and ending your Live Activity with remote push notifications</a>

```